### ISSUE TITLE
Change editor / preview windows to open in new tab by default

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1277]
If you click edit or play (once editor open) at the moment, a new window will open to display the editor/preview. You can click Ctrl-edit/play to open in a new tab instead. Should we change this so that the new tab is default instead? Main reason for asking about this is that the current windows that open don't have the normal browser bar buttons at the top (back, refresh etc.) and if links are changed so that they open in the current window rather than new tab, this makes it difficult to return to the Xerte project when previewing. I've had users here who have done this (to improve accessibility) who are then confused as they think this will also be the experience of the end users. Opening in new tab will also give people a better idea of how the project looks full screen as the default new window is smaller.

Any thoughts? @torinfo were you doing some work a while back on having the option to open in a lightbox over the workspace? This is another option too.

**Comment by ronm123** [comment]
@FayCross @torinfo I understand the issue but personally I don't think we should change the current default behaviour at least not for the editor window but probably not for the preview window either.
We have the various keyboard shortcuts but who actually knows about them?! Perhaps the edit and preview buttons should be changed to list menus instead e.g. edit and/or preview in New window, new tab, lightbox or at the very least the various keyboard options should be added to the tooltips? Certainly if we did change the current default behaviour then this would need to be the case e.g. it needs to be shown somehow what the window options are especially for a new user.
There are pros and cons to each option aren't there?
New window = how it's always been and better for keeping the editor visible while playing and previewing.
New tab = larger windows and addresses the link issue and keeps the browser buttons
Lightbox = would be good to have as an option for preview/play but not sure it would be a good idea for the editor window

**Comment by FayCross** [comment]
Yeah, we don't make the shortcuts easy to find out about. I always open editor and preview in new tabs so when I got questions about how people would get back to the project after following links I was very confused as I'd forgotten what the default view was.

I might have a look at what some other tools do when editing/previewing. I can't help but think the way we do things at the moment is just a bit dated now. Was any of this going to be looked at with the work being done on theming the workspace / editor @torinfo?

Maybe a better solution would be to just have the browser bar at the top (for preview anyway). I don't think I'd be keen on the lightbox option tbh
